### ISSUE TITLE
Add an install option for pending

### ISSUE THREAD

**Issue by thadguidry** [google/schema-dts#183]
Perhaps something like:
`npm install schema-dts-pending`

This would add the Schema.org core vocabulary inclusive of the [Pending Section](https://schema.org/docs/pending.home.html)

**Comment by thadguidry** [comment]
@Eyas hmm, but I guess a workaround or straightforward way for me to add Pending would be to simply do:

```
npm install schema-dts-gen
npx schema-dts-gen --ontology=https://schema.org/version/latest/schemaorg-all-https.jsonld
```
as mentioned on https://github.com/google/schema-dts/blob/main/packages/schema-dts-gen/README.md ?
(jsonld ending and not nt)

**Comment by Eyas** [comment]
Fixed in v1.1.2
